requirements/base.in

```diff
@@ -1,5 +1,5 @@
 # Base requirements - for all installations
-# updated May 9, 2017 for 0.11.0 release
+# updated May 27, 2020 to add CAS dependencies
```

Comment:
I'd be curious to know what is the purpose of this line since it's tracked in git.

Reply:
Good point! I think not much - should I just go ahead and remove that line?